docs/index.asciidoc

```diff
@@ -58,6 +58,17 @@ https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service] on
 Elastic Cloud. The Elasticsearch Service is available on AWS, Google Cloud
 Platform, and Microsoft Azure. {ess-trial}[Try the {es} Service for free].
 
+==== Compatibility with the Elastic Common Schema (ECS)
+
+This plugin will persist events to Elasticsearch in the shape produced by
+your pipeline, and _cannot_ be used to re-shape the event structure into a
+shape that complies with ECS.
+
```

Comment:
What would a user need to do if they need to reshape?  Or maybe your next sentence is the answer. :-)

If we can offer additional approaches or guidance on this topic, we can add more info to the LS Ref and link to it.

Reply:
A user would need to write their pipeline in such a way that they use field structure as semantically defined by the ECS schema, which is beyond the scope of the ECS Compatibility Mode of this output plugin.

An event in the Elastic Common Schema relies on the semantic meaning that ECS applies to field _values_ based on the field _names_, and as much as we'd like to magically shove everything into the schema, by the time it gets to this output we typically don't have enough information to do so (e.g., a string that looks like an ip address could be any of `client.ip`, `client.nat.ip`, `destination.ip`, `destination.nat.ip`, `host.ip`, `observer.ip`, `related.ip`, `server.ip`, `server.nat.ip`, `source.ip`, or `source.nat.ip`), and it is only in the _pipeline_ when the user who has _context_ into the data they are parsing, that mapping to the correct ECS-defined field can be done.

Reply:
I _think_ I have added some clarity in both this paragraph and the one that follows.

Reply:
Yes! Nice clarification. Thank you.